### ISSUE TITLE
Remove general time functions

### ISSUE THREAD

**Issue by rchristie** [cmlibs/zinc#71]
Zinc still has a commented-out windows implementation of the Unix gettimeofday function.
Since Zinc leaves responsibility for timing to the client application this can be removed.

**Comment by rchristie** [comment]
There is also use of the Unix times() function, currently only in commented-out old mcubes code.
These functions are in general/time.h|.cpp.

**Comment by hsorby** [comment]
Are you proposing to also remove the times() function as well?

**Comment by rchristie** [comment]
Yes I'll remove times() too as I see no reason for timing functions within Zinc. We'll use external profiler tools if needed (timing performance was the reason it was in the mcubes code).
